### ISSUE TITLE
defmt 0.2.0 - docs.rs build failed 

### ISSUE THREAD

**Issue by japaric** [knurling-rs/defmt#409]
https://docs.rs/crate/defmt/0.2.0

**Comment by jonas-schievink** [comment]
```
[INFO] [stderr]  Documenting defmt v0.2.0 (/opt/rustwide/workdir)
[INFO] [stderr] error[E0463]: can't find crate for `std`
[INFO] [stderr]   |
[INFO] [stderr]   = note: the `thumbv6m-none-eabi` target may not be installed
```
weird

**Comment by jonas-schievink** [comment]
Ah, maybe we need to tell it to not pass `--all-features`?

**Comment by Urhengulas** [comment]
This seems to be fixed, but it's not available on docs.rs. How can we update it there?

**Comment by Urhengulas** [comment]
Reading the website it seems opening an issue is the way to request a rebuild (https://docs.rs/about/builds#failures-and-rebuilds). Should we do this?

**Comment by jonas-schievink** [comment]
We can just publish 0.2.1 with this change included

**Comment by Urhengulas** [comment]
Like now, or later?

**Comment by jonas-schievink** [comment]
Feel free to do it now if you want. No other crates have to be published, just `defmt`, so it should be fairly straightforward compared to the 0.2.0 release.

**Comment by Urhengulas** [comment]
Fixed by #413 and published by releasing `defmt v0.2.1` (#428).
